referee_node/referee_resource[] resources
